components/ui/sonner.jsx

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useTheme } from 'next-themes';
+// eslint-disable-next-line import/no-self-import
```

Comment:
I don't think this is self-import. If there is a bug with eslint I'm fine leaving this line here

Reply:
It gave me a linting issue about `self-import ` when I ran the` format-check` that's the only reason I added it. 

```suggestion

```

Reply:
If this is required in order to make `format-check` pass I'm ok with resolving this comment w/o removing it

Reply:
are you sure?